src/webcomponents/helperfunctions/jsonToCol.tsx

```diff
@@ -0,0 +1,85 @@
+
+export const jsonToCol=(languages:any)=>{
```

Comment:
Do not use any types. Most of the types in this file will be inferred - just use the type for input parameter:
`export const jsonToCol = (languages: Languages) => {`

Please, use proper formatting: (prettier and eslint) https://robertcooper.me/post/using-eslint-and-prettier-in-a-typescript-project

Reply:
eslint is little buggy for my pc. can't use that

Reply:
already using prettier